release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1947,6 +1947,8 @@ This update fixes the problem by introducing client-go 1.22.
 
 * Previously, pod namespaces managed by CRI-O, for example network, IPC, and UTS, were not unmounted when the pod was removed. This resulted in leakage, driving the Open vSwitch CPU to 100%, which caused pod latency and other performance impacts. This has been resolved and pod namespaces are unmounted when removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2003193[*BZ#2003193*])
 
+* Containers are started with non-empty inheritable Linux process capabilities. This can potentially allow programs with inheritable file capabilities to elevate those capabilities to the permitted set during `execve(2)`. The workaround is to modify the entry point of a container using a utility such as `capsh(1)` to drop inheritable capabilities prior to when the primary process starts. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2076265[*BZ#2076265*])
```

Comment:
I think the point of this is that cri-o giving inheritable capabilities is atypical for a coordinator of processes on unix. Historically, users are able to elevate the privileges of a pod with capabilities, even as non-root. I don't know if we should consider it a privilege elevation

Reply:
put differently: even if the inheritable capabilities aren't specified, a user can use a step at image build to massage the file capabilities to grant the non-root user the capabilities at container run time. So even with the "fix", a non-root user can still escalate privileges. as such, dropping the inheritable capabilities doesn't do much to protect from privilege escalation, but does update the way the container proecss is setup to be more idiomatic

Reply:
Then, I suggest we take out any claims suggesting improved security from the fix. Which would leave us with the following simplified entry:

Containers are started with non-empty inheritable Linux process capabilities. The workaround is to modify the entry point of a container using a utility such as `capsh(1)` to drop inheritable capabilities prior to when the primary process starts. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2076265[*BZ#2076265*])

Are there any observations we would like to add to replace the deleted sentence?

Reply:
nope that looks great!